packages/editor/src/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { useReducer, useEffect } from 'react'
+import { Global } from '@emotion/core'
+import { ThemeProvider, Styled, ColorMode } from 'theme-ui'
+import Panel from './components/Panel'
+import editorTheme from './theme'
+import { runScript, mergeState } from './utils'
+
+const Editor = ({ panelColorMode }) => {
+  const [state, setState] = useReducer(mergeState, {
+    theme: null,
+    colorMode: null,
+  })
+
+  const getTheme = () => {
+    runScript(`window.__THEME_UI__.theme`).then(theme => {
```

Comment:
Planning on merging this as-is, but I do think it'd be better to keep the chrome-related code within the chrome extension and the editor package be a standalone UI thing – something we can address later too